pkg/controller/registry/resolver/cache.go

```diff
@@ -416,6 +416,18 @@ func WithPackage(pkg string) OperatorPredicate {
 	}
 }
 
+func WithoutDeprecatedProperty() OperatorPredicate {
+	return func(o *Operator) bool {
+		props := o.bundle.GetProperties()
```

Comment:
`nit`: `properties`, since you've already used `properties` [here](https://github.com/operator-framework/operator-lifecycle-manager/pull/1673/files#diff-4ad630e08ab9fb3fc440005ecf4cc9aaR261)

Reply:
there is only one deprecated property per bundle

Reply:
Isn't `GetProperties` returning `properties` (irrespective of whether those properties are deprecated or not)? And there's a for loop after this that's iterating on the `properties` we're getting from `GetProperties`  

```
return func(o *Operator) bool {
		properties := o.bundle.GetProperties()
		for _, property := range properties {
			if property is deprecated {
				return false
			}
		}
		return true
	}
```
reads much better

Reply:
Alternatively, you could eliminate the variable `properties` completely:
```go
for _, p := range o.bundle.GetProperties()
```

Reply:
@anik120 I thought you were talking about the function name - this makes sense to me